### ISSUE TITLE
2024-25 Playoff Matches are all "F-1"

### ISSUE THREAD

**Issue by timtim17** [ftc-scout/ftc-scout#4]
In the INTO THE DEEP season, a new playoff format was introduced (i.e. double elimination bracket). For events that used the new format, FTCLive reports the match numbers as M-1, M-2, ... M-10, ..., but FTCScout is reporting them all as F-1.

<img width="1029" alt="Screenshot 2024-12-09 at 14 25 25" src="https://github.com/user-attachments/assets/79d80b20-45f4-49e9-9382-8c67fa368d38">

**Comment by Dragon-Hatcher** [comment]
Yeah we have been trying to work out what to do about this. This is my current proposal but some others have said it isn't intuative:
![image](https://github.com/user-attachments/assets/c9850efe-6411-4b13-b4a3-2b8af8723eb6)
